src/acl/Acl.h

```diff
@@ -36,12 +36,9 @@ void RegisterMaker(TypeName typeName, Maker maker);
 /// Key comparison is case-insensitive.
 void SetKey(SBuf &keyStorage, const char *keyParameterName, const char *newKey);
 
-} // namespace Acl
-
 /// A configurable condition. A node in the ACL expression tree.
 /// Can evaluate itself in FilledChecklist context.
 /// Does not change during evaluation.
-/// \ingroup ACLAPI
 class ACL
```

Comment:
Please consider renaming the `ACL` class  to `Node` or `Condition` while we modify all these lines.
 
* The ACL name itself is not precise enough to describe what this class is about. Too many things are Access Control List-related in ACL context. We have a whole namespace with that name for them! And this class is not even a _List_ in most contexts (its invasive `next` field is _not_ used to build ACL lists for configuration directives -- we have Acl::InnerNode for that).

* The all-caps ACL spelling goes against our own CamelCase style that we do apply to abbreviations like HTTP and ICP. Many other related classes and the Acl namespace itself use that style.

* Avoiding a name spelling clash with Windows ACL typedef helps with code search and understanding.
 
If we have to change all these lines to fix Windows build, then let's address the class naming problem, so that we do not have to change them again. Renaming will add a few more changes, but it is probably worth it long-term.

I do not insist on renaming.

Reply:
done